build.xml

```diff
@@ -72,7 +72,7 @@
   <property name="jni.revision" value="1"/>
   <property name="jni.build" value="0"/> <!--${build.number}-->
   <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
-  <property name="jni.md5" value="1a6047467b59e8748f975e03016ce3d9"/>
+  <property name="jni.md5" value="4f72f2799dfee6008a386bc40afd7428"/>
```

Comment:
I have some VMs for other platforms, so I had access to Linux and Windows, 32bit and 64bit each to do native rebuilds:
- Linux 64bit: all good, committed a rebuild
- Windows 32bit: all good, committed a rebuild (done with cygwin)
- Linux 32bit: LibraryLoadTest.testLoadJAWT only worked after setting LD_LIBRARY_PATH to $JAVA_HOME/lib/i386. Otherwise all good, committed a rebuild.

- Windows 64bit: LibraryLoadTest.testLoadJAWT and LibraryLoadTest.testLoadLibraryWithLongName fail. I couldn't get them to work. They are independent of my change but I don't feel comfortable committing the native rebuild. Maybe the problem is that I built it with cygwin instead of MSVC?

So I committed 3 natives. I'd be happy if you could take care of the others.